<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Micro_PlasticAnalyzerClass</class>
 <widget class="QMainWindow" name="Micro_PlasticAnalyzerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>812</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Micro-PlasticAnalyzer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,0">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="imgLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="inputLabel">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Input&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImg">
            <property name="text">
             <string>Image/Video</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CameraCombo"/>
          </item>
          <item>
           <widget class="QComboBox" name="InputCamCombo">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCam">
            <property name="text">
             <string>Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="convertLabel">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;ConvertColor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnConColor">
              <property name="text">
               <string>GrayScale</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="OutputverticalLayout">
            <item>
             <widget class="QLabel" name="outputLabel">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Algorithms&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="outputCombo">
              <item>
               <property name="text">
                <string>NULL</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Inversion</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Algebra</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blurring</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sharpening</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Canny</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Threshold</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Adaptive Threshold</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Histogram Equalization</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Opening</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Closing</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tophat</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blackhat</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Contour</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Watershed</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Integral</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Distance Transform</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2-Pass(4connectivity)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2-Pass(8connectivity)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Micro-Plastic Detection</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QSlider" name="outputSlider">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>-128</number>
                </property>
                <property name="maximum">
                 <number>127</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBelow</enum>
                </property>
                <property name="tickInterval">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sliderSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>-128</number>
                </property>
                <property name="maximum">
                 <number>127</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QSlider" name="outputSlider_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>-255</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBelow</enum>
                </property>
                <property name="tickInterval">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sliderSpinBox_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>-255</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="OstucheckBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Ostu</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="AveragecheckBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Average</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>201</width>
               <height>91</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="morphLabel">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Morphology&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioRect">
                <property name="text">
                 <string>Rect</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioCross">
                <property name="text">
                 <string>Cross</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioEllipse">
                <property name="text">
                 <string>Ellipse</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="passMinlabel">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Min Size&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="passMinspinBox">
              <property name="maximum">
               <number>20000</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="passMaxlabel">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Max Size&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="passMaxspinBox">
              <property name="maximum">
               <number>20000</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="passFiltercheckBox">
              <property name="text">
               <string>Filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="passRectcheckBox">
              <property name="text">
               <string>Rect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="passOrigincheckBox">
              <property name="text">
               <string>Origin</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="passlabel">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Minimum Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="passMinInfolabel">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="passlabel2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="passInfolabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>199</width>
            <height>198</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSet">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnStartPause">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>▶</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="videoSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Qt_image_Camera.qrc"/>
 </resources>
 <connections/>
</ui>
